### DISCUSSION TITLE
【旭日图 】V5.0 如何实现 v2.0版本的语法:  chart.polygon().position(Stat.tree.rect('children'))

### DISCUSSION THREAD

**Discussion by Wennier** [antvis/G2#5367] [Q&A]
想要实现v2版本demo中的这种旭日图：
<img width="384" alt="image" src="https://github.com/antvis/G2/assets/21079822/31f6426b-2765-4b3c-bec2-79ff4c926363">

但是demo中有些语法V5中已经不支持。

比如Stat.tree.rect('children')、.position(Stat.tree.rect('children'))

想问一下如何使用V5的语法实现上面的代码功能。

**Comment by pearmini** [comment]
可以提供一下需要可视化的数据吗？

**Reply by pearmini** [reply]
可以参考这个讨论：https://github.com/antvis/G2/discussions/5323
